### ISSUE TITLE
Speaker detection with zipformer

### ISSUE THREAD

**Issue by AlexandderGorodetski** [k2-fsa/icefall#1453]
Hello guys. 

In our lab we have inhouse ADOS database which contains audio of children, therapists and parents. The recordings are very noisy. 

We have to build system which will detect speech of child, therapist and parent.

We saw some works for speaker recognition/diarization with a conformer. In order to get some novelty we would like to use Zipformer for this task. Do you think we can use Zipformer for this purpose? Do you think we can train Zipformer with cross-entropy loss? Of course I mean using of Encoder only (without decoder and without joiner).

Thanks,
AlexG.

**Comment by marcoyang1998** [comment]
Hi Alex, I've trained a Zipformer for speaker verification, the training loss is cross-entropy. It worked well.
